dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderImpl.java

```diff
@@ -82,6 +91,9 @@
 
 	/** Configuration for the ACME client. */
 	private static AcmeConfig acmeConfig;
+	
+	@Reference
+	private com.ibm.wsspi.kernel.service.location.WsLocationAdmin wslocation;
```

Comment:
Is the kernel component already in our bnd.bnd file?

Reply:
yes